lib/Ohmbrewer_Screen.h

```diff
@@ -213,6 +213,11 @@ namespace Ohmbrewer {
              */
             Menu* _homeMenu;
 
+           /**
```

Comment:
Actually, we shouldn't need to have an instance variable for the main menu. We only need to keep track of the root (HomeMenu) and the current menu. I think that's already implemented...